src/main/kotlin/com/openlattice/chronicle/controllers/StudyController.kt

```diff
@@ -258,8 +259,54 @@ class StudyController @Inject constructor(
         return if (retrieve) studyService.getStudy(studyId) else null
     }
 
-    override fun destroyStudy(studyId: UUID) {
-        TODO("Not yet implemented")
+    @Timed
+    @DeleteMapping(
+        path = [STUDY_ID_PATH],
+        produces = [MediaType.APPLICATION_JSON_VALUE],
+    )
+    override fun destroyStudy(@PathVariable studyId: UUID): UUID {
+        accessCheck(AclKey(studyId), EnumSet.of(Permission.OWNER))
+        val currentUser = Principals.getCurrentSecurablePrincipal()
+        logger.info("Deleting study with id $studyId")
+        // val currentUserEmail = getUser(managementApi, Principals.getCurrentUser().id).email
+        val deleteStudyDataJob = ChronicleJob(
+            id = idGenerationService.getNextId(),
+            contact = "test@openlattice.com",
+            definition = DeleteStudyUsageData(studyId)
+        )
+        return storageResolver.getPlatformStorage().connection.use { conn ->
+            AuditedOperationBuilder<UUID>(conn, auditingManager)
+                .operation { connection ->
+                    var jobId = chronicleJobService.createJob(connection, deleteStudyDataJob)
+                    logger.info("Created job with id = $jobId")
+                    studyService.deleteStudies(connection, listOf(studyId))
+                    studyService.deleteStudiesFromOrganizations(connection, listOf(studyId))
```

Comment:
I think you also need to delete from the `study_participants` table.

Reply:
done